apps/expo/src/components/tracker/set-entry.tsx

```diff
@@ -0,0 +1,36 @@
+import { Switch, TextInput, View } from 'react-native'
+import { Feather } from '@expo/vector-icons'
+
+import Button from '~/components/ui/button/button'
+
+const SetEntry: React.FC = () => {
+  return (
+    <View className="flex flex-row items-center justify-between px-2">
+      <View
+        className="flex basis-14 flex-row justify-start py-1"
+        testID="set-type-button-container"
+      >
```

Comment:
switch here (cough)

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/ea36a10b-17ec-45db-a4d5-de480ddc764d)


one more thing is the text being black when input